### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nemivant**

### CODE REVIEW

**Comment by nemivant** [comment]
@horis233 Please review and update the readme. 

Do you have any information for these sections? If you don't, you can delete these sections. 

### Quick start guide

Use the following quick start commands for building and testing the operator:
```
<!--- Include any required quick start commands -->
```

### Debugging guide

Use the following commands to debug the operator:

```
<!--- Include any required quick start commands -->
```

File: deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -37,50 +37,39 @@ spec:
       kind: CommonService
       name: commonservices.operator.ibm.com
       version: v3
-  description: |-
-    This operator is a bridge to connect CloudPaks and ODLM/Common Services, and it can be also installed in standalone mode.
+  description: The `ibm-common-service-operator` is a bridge to connect IBM Cloud Paks and Operand Deployment Lifecycle Manager (ODLM) with IBM Cloud Platform Common Services. You can also install the `ibm-common-service-operator` in stand-alone mode.
```
Comment (chenzhiwei) [inline deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml:40]: you should use block indicator here(`description: |-`), see: https://yaml.org/spec/1.2/spec.html#id2794534

File: deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -37,50 +37,39 @@ spec:
...
-            - name: ibm-helm-repo-operator
-            - name: ibm-catalog-ui-operator
-          registry: common-service
-    ```
+    ## Supported platforms
+
+    Red Hat OpenShift Container Platform 4.3 or newer installed on one of the following platforms:
+
+       - Linux x86_64
+       - Linux on Power (ppc64le)
```
Comment (chenzhiwei) [inline deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml:57]: we also support Z (s390x)

Comment (nemivant) [reply deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml:58]: @chenzhiwei Added Z.

File: deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -38,49 +38,40 @@ spec:
       name: commonservices.operator.ibm.com
       version: v3
   description: |-
```
Comment (nemivant) [inline deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml:40]: added `|-`

**Comment by chenzhiwei** [comment]
@DanielXLee Could you manually upload this CSV to your own testing quay and verify if it works or not?

**Comment by DanielXLee** [comment]
@chenzhiwei verified in my quay.io, running well.

**Pull request merged**